core/lib/refinery/crud.rb

```diff
@@ -283,7 +283,8 @@ def reorder
             # Based upon https://github.com/matenia/jQuery-Awesome-Nested-Set-Drag-and-Drop
             def update_positions
               previous = nil
-              params[:ul].each do |_, list|
+              # TODO: Find fix because `to_unsafe_h` badly reorder the hash
```

Comment:
@sjoulbak i don't know if it's still an up-to-date bug but it's probably why the following specs failed: https://github.com/refinery/refinerycms/pull/3122/files#diff-b2b23769a8e7837f008b49da20475dc5R125